packages/main/src/invitationRoutes.ts

```diff
@@ -25,6 +27,25 @@ export const addInvitationRoutes = async (app: express.Express, agent: ServiceAg
         return
       }
 
+      const connRecord = shortUrlRecord?.getTag('relatedFlowId') as string
```

Comment:
Move to (req.accepts('json')) { } in order to be only catched when Hologram App actually scans the QR code (otherwise, it might be possible that a web browser does that, e.g. in [http:/](https://hologram.zone/?_url=aHR0cHM6Ly9hLnVuaWMtaWQtdmVyaWZpZXIuZGVtb3MuZGV2LjIwNjAuaW8vcz9pZD0wMzVjNjVjNC0yZDM5LTRlMDQtOGFkMC05MDUwMDFhZmFmMjU))